spec/models/like_spec.rb

```diff
@@ -0,0 +1 @@
+require 'rails_helper'
```

Comment:
Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping. (https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)